src/main/groovy/com/morpheusdata/scvmm/ScvmmApiService.groovy

```diff
@@ -1610,8 +1610,9 @@ foreach (\$network in \$networks) {
                     // the expected count.. we are good
                     log.debug "serverStatus: ${serverDetail.server?.Status}, opts.dataDisks: ${opts.dataDisks?.size()}, additionalTemplateDisks: ${opts.additionalTemplateDisks?.size()}"
 
-                    if (serverDetail.server?.Status != 'UnderCreation' &&
-                            serverDetail.server?.VirtualDiskDrives?.size() == 1 + ((opts.dataDisks?.size() ?: 0) - (opts.additionalTemplateDisks?.size() ?: 0))) {
+                    if (serverDetail.server?.Status != 'UnderCreation') {
+                    // if (serverDetail.server?.Status != 'UnderCreation' &&
```

Comment:
Since the additionalTemplateDisks are showing up properly, can we re-enable this condition ?